samples/Client/DaprClient/Readme.md

```diff
@@ -2,177 +2,54 @@
 The client sample shows how to make Dapr calls to publish events, save state, get state and delete state using a Dapr client apis. 
 
 ## Prerequistes
-* [.Net Core SDK 3.1](https://dotnet.microsoft.com/download)
+
+* [.Net Core SDK](https://dotnet.microsoft.com/download)
 * [Dapr CLI](https://github.com/dapr/cli)
 * [Dapr DotNet SDK](https://github.com/dapr/dotnet-sdk)
 
-
  ## Running the Sample
 
  To run the sample locally run this command in DaprClient directory:
  ```sh
- dapr run --app-id DaprClient -- dotnet run
+ dapr run --app-id DaprClient -- dotnet run <sample number>
  ```
 
- Above command will run Dapr runtime and launch the app and will show logs both form Dapr runtime and the application. The client app will make calls to Dapr runtime to publish events, save state, get state and delete state using the gRPC client.
- Logs from application will show following in the command window:
-```sh
-Published Event!
-Saved State!
-Got State: small  yellow
-Deleted State!
-Executing transaction - save state and delete state
-Executed State Transaction!
-State not found in store
-Retrieved state: small  yellow with etag: 1
-Saved modified state : small  orange
-Retrieved state: small  orange with etag: 2
-Saved state with old etag: 1 successfully? False
-Saved state with latest etag: 2 successfully? True
-Published deposit event!
-Done
+Running the following command will output a list of the samples included. 
+ ```sh
+ dapr run --app-id DaprClient -- dotnet run
  ```
 
-To invoke RoutingService via HTTP, run [RoutingService](../../AspNetCore/RoutingSample) or to invoke GrpcService via gRPC, run [GrpcService](../../AspNetCore/GrpcServiceSample) sample first. Then, invoke the client using the corresponding command below:-
-
-``` sh
- dapr run --app-id DaprClient -- dotnet run --useRouting true
-```
-
-or
+ Press Ctrl+C to exit, and then run the command again and provide a sample number to run the samples.
 
-``` sh
- dapr run --app-id DaprClient -- dotnet run --useGrpcsample true
-```
+  ```sh
+ dapr run --app-id DaprClient -- dotnet run 0
```

Comment:
Remove this or maybe add a line: "for example, run this command to invoke the grpc sample"